VTUL/VIVO_VT_Theme
# Vivo Rebranding

**Rebrand to conform to the VT Unirel guidelines (more or less).**
* * *

**JIRA Ticket**: (https://webapps.es.vt.edu/jira/browse/LIBTD-1693) (:star:)

# What does this Pull Request do? (:star:)
Implement the VT header and body styles (STILL NEED TO IMPLEMENT FOOTER)

# What's the changes? (:star:)
* Adds new html, css rules, and js scripts

# How should this be tested?
* Check that the site looks like this:
<img width="1366" alt="screen shot 2019-02-07 at 4 00 53 pm" src="https://user-images.githubusercontent.com/1202435/52442508-4e42db00-2af1-11e9-9b4e-d17f10bb6b85.png">
* Check that the site is still functional

# Additional Notes:
* branch: `LIBTD-1693`

# Interested parties
@yinlinchen 

(:star:) Required fields
